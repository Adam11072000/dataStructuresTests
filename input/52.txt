init 4915
addImage 4977
addImage 4983
addImage 4917
addImage 4983
mergeSuperPixels 4983 1276 3006
resetLabelScore 4977 1263 0
deleteImage 4917
mergeSuperPixels 4983 360 0
addImage 3663
getHighestScoredLabel 4977 249
resetLabelScore 5314 2446 3170
quit