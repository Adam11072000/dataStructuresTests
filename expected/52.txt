init done.
addImage: SUCCESS
addImage: SUCCESS
addImage: SUCCESS
addImage: FAILURE
mergeSuperPixels: SUCCESS
resetLabelScore: INVALID_INPUT
deleteImage: SUCCESS
mergeSuperPixels: SUCCESS
addImage: SUCCESS
getHighestScoredLabel: FAILURE
resetLabelScore: FAILURE
quit done.
